table/block_based/partitioned_filter_block.cc

```diff
@@ -370,11 +368,7 @@ void PartitionedFilterBlockReader::CacheDependencies(bool pin) {
         }
       }
     }
-#ifndef NDEBUG
-    if (!s.ok()) {
-      TEST_SYNC_POINT("FilterReadError");
-    }
-#endif
+    IGNORE_STATUS_IF_ERROR(s);
```

Comment:
I believe this should be inside the loop since any single `MaybeReadBlockAndLoadToCache` call might fail due to fault injection. (The way it's written, it only checks the final `Status`.)

Reply:
... that is, you can do what you do in `PartitionedIndexReader` below :)

Reply:
Its inside the loop at line 352.

Reply:
Sorry, I misread that.